versions/3.1.1.md

```diff
@@ -314,12 +314,22 @@ This is the root object of the [OpenAPI document](#openapi-description).
-| <a name="oas-security"></a>security                    |  [[Security Requirement Object](#security-requirement-object)]  | A declaration of which security mechanisms can be used across the API. The list of values includes alternative security requirement objects that can be used. Only one of the security requirement objects need to be satisfied to authorize a request. Individual operations can override this definition. To make security optional, an empty security requirement (`{}`) can be included in the array.                                                                                                                                               |
+| <a name="oas-security"></a>security                    |  [[Security Requirement Object](#security-requirement-object)]  | See [the `security` field](#oas-security-field).
 | <a name="oas-tags"></a>tags                            |                  [[Tag Object](#tag-object)]                   | A list of tags used by the document with additional metadata. The order of the tags can be used to reflect on their order by the parsing tools. Not all tags that are used by the [Operation Object](#operation-object) must be declared. The tags that are not declared MAY be organized randomly or based on the tools' logic. Each tag name in the list MUST be unique.                                                                                                                                                                               |
 | <a name="oas-external-docs"></a>externalDocs            | [External Documentation Object](#external-documentation-object) | Additional external documentation.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
 
 This object MAY be extended with [Specification Extensions](#specification-extensions).
 
+###### <a name="oas-security-field"></a>The `security` field
+
+The `security` field describes how requests are authorized. If omitted, then requests are always authorized; otherwise, it is an array of security requirement objects, only one of which needs to be satisfied for the request to be authorized.
+
+Because the empty [security requirement object](#security-requirement-object) `{}` will always authorize the request, it follows that `security: [{}]` means the same as omitting `security` entirely: namely, all requests are authorized.
```

Comment:
I think "no info" means "no info", see above.

Reply:
Could we perhaps be a little more precise with the language here, please? When you say that X means Y, should that be interpreted as:

1. "Personally, I have always taken X as implying Y" (i.e. a reference to an existing spec)
2. or: "I believe that the majority of people using OpenAPI have always taken X as implying Y" (existing spec)
3. or: "I would like X to imply Y" (i.e. with reference to a dev/future spec)

(1) doesn't need any rationale; if that's your interpretation, then that's your interpretation. Nobody can deny that.

(2) _may_ benefit from supporting evidence (depending on how controversial it seems, for example)

(3) may well benefit from a supporting argument, especially if "X implies Y" is not clearly true in the existing spec.

So which is it?

Reply:
For a long comment (by me), that sure was a waste of space. Sorry, ignore me. I'm not sure where I was going with that.